### PR TITLE
Ref#43673 make order and modifify model order,order details

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43673](https://edu-redmine.sun-asterisk.vn/issues/43673)

## WHAT this PR does?
make order and modifify model order,order details

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
Ready

**Comment by hoangtrungthong** [comment]
ready

File: resources/views/user/history_order.blade.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                                                <td class=" px-6 py-3 whitespace-nowrap text-right text-sm font-medium">
+                                                    <form action="{{ route('user.destroyOrder', $order->id) }}" method="post">
+                                                @csrf
+                                                @method('DELETE')
+                                                {{-- <a href="{{ route('admin.products.show', $product->slug) }}" class="inline-block bg-indigo-500 hover:bg-indigo-700 text-white text-center py-1 px-3 rounded">
+                                                    <i class="fas fa-eye"></i>
+                                                </a>
+                                                <a href="{{ route('admin.products.edit', $product->slug) }}" class="inline-block bg-yellow-500 hover:bg-yellow-700 text-white text-center py-1 px-3 rounded">
+                                                    <i class="fas fa-edit"></i>
+                                                </a> --}}
```
Comment (ghost) [inline resources/views/user/history_order.blade.php]: ko dùng xóa đi chứ

**Comment by ghost** [comment]
commented

**Comment by hoangtrungthong** [comment]
ready

**Pull request merged**